src/store/reducer.ts

```diff
@@ -43,7 +44,8 @@ const initialState: State = {
   offers: [],
   authentication: {
     timestamp: 0
-  }
+  },
+  selectedFacilityId: undefined
```

Comment:
it is better to not initialize this property. This is the same as initialize it with `undefined` value